src/databricks/labs/ucx/source_code/known.json

```diff
@@ -5149,6 +5149,24 @@
     "johnsnowlabs.visual": [],
     "johnsnowlabs.viz": []
   },
+  "jsonschema": {
+    "jsonschema": [],
+    "jsonschema._format": [],
+    "jsonschema._keywords": [],
+    "jsonschema._legacy_keywords": [],
+    "jsonschema._types": [],
+    "jsonschema._typing": [],
+    "jsonschema._utils": [],
+    "jsonschema.cli": [],
+    "jsonschema.exceptions": [],
+    "jsonschema.protocols": [],
+    "jsonschema.tests": [],
```

Comment:
Remove

Reply:
What would be the rationale for removing these modules? We're adding them to improve linting speed.

Reply:
I think we should aim to keep the `know.json` relevant to the code that users use so that it does not blow up. The same for hidden modules, a user can import it, but (s)he probably does not.

Reply:
Indeed it doesn't hur,  `known.json` is only loaded once. `jsonschema` is a very popular package, see statistics here: https://pypi.org/project/jsonschema/